### DISCUSSION TITLE
Quality of Life Suggestions

### DISCUSSION THREAD

**Discussion by Boomer42** [FakeFishGames/Barotrauma#5684] [Ideas and suggestions]
### **General Quality of Life Ideas**

1. Storage for depth charges
What a mess, a shelf for for 4 - 6 depth charges would be nice.
![image](https://user-images.githubusercontent.com/8073774/116820739-e901b700-ab4c-11eb-8ec7-2eb06c400480.png)

2. Add more wire connections to the Engine
Add 2 for the small and maybe 3 for the bigger one. Making the whole sub a daisy chain of boxes can cripple the ship.
![image](https://user-images.githubusercontent.com/8073774/116820960-01be9c80-ab4e-11eb-8af9-3aaacfde2fc9.png)

3. Add more wiring points to the Junction boxes.
Five points of wiring per box is not enough once you daisy chain them together you already lost two wiring points.
Either add daisy chain points or make it six wires so you can have four devices per box instead of three. 
This could of been a conscious design choice, and this is just my opinion.
![image](https://user-images.githubusercontent.com/8073774/116821085-d0929c00-ab4e-11eb-8647-43afe153e468.png)

4. Visual bar to see how much ammo is left in boxes
![image](https://user-images.githubusercontent.com/8073774/116821141-0d5e9300-ab4f-11eb-97a0-27c25931e5de.png)

5. Be able to choose what items spawn at the cargo points
Have a general cargo spawn point, and have more specific ones. For example if you want ammo you bought spawn in you ammo storage room.
![image](https://user-images.githubusercontent.com/8073774/116821230-6af2df80-ab4f-11eb-984f-16b353135878.png)

6. be able to change the cone illumination on the turrets and spot lights
In either in the editor or in game, but has a range penalty
![image](https://user-images.githubusercontent.com/8073774/116822006-eb670f80-ab52-11eb-9b2b-e785a1cf94c8.png)

7. Bug, I placed small flour pots and I cant pick them up.....
Improve allowable placements 
![image](https://user-images.githubusercontent.com/8073774/116822096-4d277980-ab53-11eb-8808-a942a30e6c68.png)

8. Gun Flash Lights
Yeah I know its a mod but I think it should be in the default game

9. A Writable billboard with permissions, that shows a couple lines in game but opens to a log book for useful info you wrote.

10. Refillable Fire extinguishers, instead of making new ones

### **AI Ideas**

11. Be able to tag cabinets on what is stored here for when the AI cleans up items. Have a priority on what items to clean first

13. Be able to choose a passive job if he can't perform his main task, right now they keep running the engine without orders

14. The orders stay on the AI between outpost visits, right now you got re-order them every time you leave an outpost

15. Allow us to interact with inventory of AI, without having to taze or stun them and then scramble to change things out before they wake up

16. Allow security AI to replenish their gun ammunition themselves and set how much ammo they can to hold

17. Orders to not use diving suits if you don't have enough for players

### **Gameplay Ideas**

18. Damageable air vents and needing repair

19. The helm stat updates to fast vs the others, also the helm stat could help you see more on sonar as you get better

20. Way to reload SMG Mags without making new ones

### **21.  Be able to load in new subs to buy in an existing campaign without restarting!**
As a Sub builder this is aggravating...

**Comment by frijoles108** [comment]
I thought, considering how much of your suggestions I agreed with, I would add a couple of my own rather than just making a seperate post:
- allow us to interact with inventory of ai, without having to taze or stun them and then scramble to change things out before they wake up. 
- Allow security AI to replenish their gun ammunition themselves, or just add the above feature so we give them a number of ammunition to work with (probably better idea so they aren't trigger happy burning through supply)

**Comment by Boomer42** [comment]
I never thought about accessing AI Inventory that's a good point, we usually avoid using AI except for as a quick a dirty medic.
Ill add these to the list you don't mind I like this Ideas also.

Also from blowing up a few times from fire I notice you can reload ammo crates and welding tanks, that's awesome! but they missed Fire extinguishers you have remake them instead of refillable.
